tests/FunctionalDeprecated/Comment/CommentTest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sabberworm\CSS\Tests\FunctionalDeprecated\Comment;
+
+use PHPUnit\Framework\TestCase;
+use Sabberworm\CSS\Comment\Comment;
+
+/**
+ * @covers \Sabberworm\CSS\Comment\Comment
+ */
+final class CommentTest extends TestCase
+{
+    /**
+     * @test
+     */
+    public function toStringRendersCommentEnclosedInCommentDelimiters(): void
```

Comment:
Why is this method being moved here to test something that's deprecated?  Shouldn't the introduction of the test class be a separate PR?

Reply:
I'm moving this one test from `Tests/Functional/` to `Tests/FunctionalDeprecated/`, where tests for deprecated functionality should be placed. (This will allow us to have different PHPUnit configuration files later on, allowing calls to deprecated functionality in `Tests/FunctionalDeprecated/`, but not in  `Tests/Functional/`.)